### ISSUE TITLE
Version 2.3.0 on npm seems incomplete

### ISSUE THREAD

**Issue by Morendil** [kRITZCREEK/pscid#35]
The version of pscid currently published to the npm registry seems non-functional, it's [missing](https://www.jsdelivr.com/package/npm/pscid) `index.js` and source files. (I'm unfamiliar with the workings of npm so it took me a while to figure this out and I might still be confused as to what exactly is going on.) Forcing 2.0.2 worked.

**Comment by kritzcreek** [comment]
Thank you for the report, I'll look into it!

**Comment by kritzcreek** [comment]
Should be fixed in 2.3.1, could you confirm?

**Comment by Morendil** [comment]
Hmm, I *am* seeing an `index.js` and `node` can apparently run it, but I get the following error:
`: No such file or directory`

**Comment by kritzcreek** [comment]
Could you try uninstalling whatever version you have sitting around? I've tried it on two computers now and 2.3.1 works on both....

**Comment by Morendil** [comment]
This might be an issue with the registry or with packaging, I'm not getting the same results with `yarn` as I am with `npm`. My packaging tool of choice is yarn.

Here's what I'm doing:
```
mkdir pscid-test
docker run --rm -it -v $pwd:/app -w /app node /bin/bash
```

That is, creating what should amount to a clean system with the latest Node. Then:
```
yarn add pscid
yarn run pscid
```

And the result is as before:
```
yarn run v1.3.2
warning package.json: No license field
$ /app/node_modules/.bin/pscid
: No such file or directory
error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Using `yarn global add` doesn't help.

With `yarn add pscid@2.0.2` the install works and I get the following output:
```
ERROR:
I couldn't find any source directories to watch when trying app/, src/, test/ and tests/.
```
etc. - this is what I expect.

**Comment by htmue** [comment]
Same issue on my computer with 2.3.1:
```
% pscid
env: node\r: No such file or directory
```
After changing the line endings in `index.js` from DOS to Unix it works.

**Comment by kritzcreek** [comment]
Oh my... is it a line ending problem? I'll see if I can fix that.

**Comment by kritzcreek** [comment]
So I built the 2.4.0 release on a Linux machine instead of Windows, can you check if that fixes things? Thanks!

**Comment by htmue** [comment]
Yes, that did the trick, thanks a lot! ...and thanks for the very useful tool, btw, I use it every day.

**Comment by kritzcreek** [comment]
Glad it's working well for you. So I guess I'll have to remember to publish these releases from a Linux machine. Thanks for confirming!

**Comment by rnons** [comment]
I'm seeing this again on v2.9.0

```
env: node\r: No such file or directory
```

Workaround is mentioned in https://github.com/kritzcreek/pscid/issues/35#issuecomment-367631400
> After changing the line endings in `index.js` from DOS to Unix it works.

**Comment by kritzcreek** [comment]
Sorry I forgot to have to release from Linux. Fixed in v2.9.1 I hope
